vortex-array/src/arrays/decimal/mod.rs

```diff
@@ -44,8 +44,8 @@ impl VTable for DecimalVTable {
 #[derive(Clone, Debug)]
 pub struct DecimalEncoding;
 
-/// Maps a decimal precision into the small type that can represent it.
-pub fn precision_to_storage_size(decimal_dtype: &DecimalDType) -> DecimalValueType {
+/// Maps a decimal precision into the smallest type that can represent it.
+pub fn smallest_storage_type(decimal_dtype: &DecimalDType) -> DecimalValueType {
```

Comment:
Any reason this isn't a function on DecimalDType?

Reply:
It's an implementation detail